### ISSUE TITLE
Write STARK verifier, and test it

### ISSUE THREAD

**Issue by Sword-Smith** [Sword-Smith/twenty-first#10]
The STARK verifier has been implemented in a test, but we need a separate function for it.

**Comment by Sword-Smith** [comment]
Merge into master once this is done.

**Comment by Sword-Smith** [comment]
Function has been written but no good test exists for it yet.
